bolt/lib/Core/BinaryContext.cpp

```diff
@@ -120,6 +120,7 @@ void BinaryContext::logBOLTErrorsAndQuitOnFatal(Error E) {
   });
 }
 
+std::mutex ArchMutex;
```

Comment:
Maybe make it static in Relocation?
@aaupov WDYT?

Reply:
Move Relocation::Arch assignment out into RewriteInstance.cpp just above `createBinaryContext`?

Reply:
Do we still need this?

Reply:
> Do we still need this?

We don't need the comment, marked as resolved.